cadc-util/src/main/java/ca/nrc/cadc/auth/BearerTokenPrincipal.java

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
This isn't the full header comment that includes the license terms. You can find the correct one in AuthenticationUtil.